src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
-package core.basesyntax;
+package core.colors;
```

Comment:
recommendation: move all classes back to basesyntax for now
later we will see how we can store classes in different packages